deb/sysbox-ce/sysbox-ce.postinst

```diff
@@ -85,7 +85,8 @@ function is_wsl() {
 function enable_unprivileged_userns() {
 
     local val=$(sysctl kernel.unprivileged_userns_clone)
-    if [[ "${val##* }" = 0 ]]; then
+    local exists=$?
```

Comment:
While this works, I think it mixes up "file existence" with "I got an error", which are now necessarily the same thing.

How about something like:

```
if [ -f "/proc/sys/kernel/unprivileged_userns_clone" ]; then
  local val=$(sysctl kernel.unprivileged_userns_clone)
  if [[ "${val##* }" = 0 ]]; then
    sysctl -w kernel.unprivileged_userns_clone=1 >/dev/null 2>&1
  fi
fi
```

Reply:
Makes sense, I changed it to address that.